concepts/05 Widgets/DataGrid/00 Getting Started with DataGrid/80 Export Data to Excel.md

```diff
@@ -0,0 +1,216 @@
+DevExtreme includes a static method [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) that exports grid data to the Excel format. Its minimum configuration requires a **DataGrid** instance and an Excel worksheet to which grid data should be exported. After the export, you should save the data to an XLSX file.
```

Comment:
>you should save the data to an XLSX file

But what if we don't want to use XLSX?
![image](https://user-images.githubusercontent.com/33040934/95571079-8c17a580-0a30-11eb-930e-ec08d9d8f04b.png)

Reply:
```suggestion
DevExtreme includes a static method [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) that exports grid data to the Excel format. Its minimum configuration requires a **DataGrid** instance and an Excel worksheet to which grid data should be exported. After the export, you should save the data to an Excel file.
```

Reply:
LGTM